deploy-disconnected-registry/ocp-sync.sh

```diff
@@ -82,10 +84,12 @@ if [[ ${1} == 'hub' ]]; then
     trust_internal_registry 'hub'
 
     if ! ./verify_ocp_sync.sh 'hub'; then
-        oc create namespace ${REGISTRY} -o yaml --dry-run=client | oc apply -f -
-
-        export REGISTRY_NAME="$(oc get route -n ${REGISTRY} ${REGISTRY} -o jsonpath={'.status.ingress[0].host'})"
-        ${PODMAN_LOGIN_CMD} ${DESTINATION_REGISTRY} -u ${REG_US} -p ${REG_PASS} --authfile=${PULL_SECRET} # to create a merge with the registry original adding the registry auth entry
+        if [[ ${CUSTOM_REGISTRY} == "false" ]]; then
+            oc create namespace ${REGISTRY} -o yaml --dry-run=client | oc apply -f - 
+            # export REGISTRY_NAME="$(oc get route -n ${REGISTRY} ${REGISTRY} -o jsonpath={'.status.ingress[0].host'})"
```

Comment:
Just to ensure: Adding the common.sh you get this var instead of exporting again, right?